clang/lib/CodeGen/CGLoopInfo.cpp

```diff
@@ -476,6 +476,23 @@ EmitIVDepLoopMetadata(LLVMContext &Ctx,
   LoopProperties.push_back(MDNode::get(Ctx, MD));
 }
 
+/// Setting the legacy LLVM IR representation of the ivdep attribute
+static void EmitLegacyIVDepLoopMetadata(
+    LLVMContext &Ctx, llvm::SmallVectorImpl<llvm::Metadata *> &LoopProperties,
+    const LoopAttributes::SYCLIVDepInfo &I) {
+  // Emit "enable" metadata upon the infinite safelen specification
```

Comment:
Can you reword this one?  The 'upon the infinite safelen specification' doesn't really parse for me.

Reply:
Done - please check if the comment makes sense now.